packages/datadog_flutter_plugin/android/src/test/kotlin/com/datadoghq/flutter/DatadogSdkPluginTest.kt

```diff
@@ -74,7 +70,7 @@ class DatadogSdkPluginTest {
 
     @AfterEach
     fun afterEach() {
-        plugin.stop()
+        Datadog.stopInstance()
```

Comment:
there is a problem with `stopInstance`, that it doesn't clean up RUM monitors/tracers. Not sure if it can affect your tests though.

Reply:
Not these no, as they don't actually send data.

But, the Flutter integration tests are split up in a way to force an app shutdown and rebuild, which is obviously slower. It would be nice if there was a way to shutdown and restart everything if just for those tests.